.github/workflows/maestro_release.yaml

```diff
@@ -100,16 +100,29 @@ jobs:
           generate_release_notes: true
           name: ${{ needs.build.outputs.release_name }} – ${{ github.ref_name }}
 
-  open-followup-pr:
+  followup-pr:
     needs: [build, release]
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
+
```

Comment:
nit: remove the extra blank lines between steps (you added them above/below the new steps) for consistent style